pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -234,3 +254,84 @@ format_annot <- function(annot_list) {
   rownames(annot) <- annot$input
   return(annot)
 }
+
+# Fix annotations makes annotations compatible between samples with different types.
+# The possible options for features_types at this stage are 0,1,2
+#  0 is SYMBOL/ SYMBOL
+#  1 is IDS/SYMBOL
+#  2 is IDS/IDS
+equalize_annotation_types <- function(annot_list, counts_list, features_types_list, samples) {
```

Comment:
Why use numbers and not strings? "sym-sym", "id-sym", "id-id" would be clearer.

Reply:
also, roxygen2 docs!

Reply:
I went with numbers so they can be generated by adding bools, but feel free to change it!

Reply:
where could it be useful to generate them as bools? I'd probably go also with strings

Reply:
changed the hardcoded ints to strings defined in sysdata.R